en-US/Players_Guide/03_Creating_a_Character/Cultures/Cultures/Wood_Elf.md

```diff
@@ -1,4 +1,29 @@
 #### Wood Elf
 
-> **Warning:**
-> This culture has not yet been transferred from the Google Doc.
+Deeply attuned to the forest, this culture prizes speed, precision, and harmony with nature.
+Its members are skilled hunters, trackers, and guardians of the wild.
+\
+\
+**Fleet of Foot.**
+Your Speed increases by 5 feet.
+\
+**Nature’s Ally.**
+You can cast [Animal Friendship](#Animal_Friendship_animal_friendship) without material components once per long rest.
+Your spellcasting ability for this spell is Intelligence, Wisdom, or Charisma (whichever is highest).
+\
+**Nature’s Touch.**
+Choose one of the following.
```

Comment:
Is it clear enough, that you do this once at character creation? I _think_ it is (which is why I didn't suggest a change), but I'd like your opinion on the matter.